spec/menu_spec.rb

```diff
@@ -0,0 +1,21 @@
+require 'menu'
+
+describe Menu do
+
+  it 'creates a new instance of menu' do
+    expect(subject).to be_instance_of Menu
+  end
+
```

Comment:
you could think about using mocks/doubles here to test the behaviour of the .menu method instead of using the actual menu to test, which I think falls more into testing behaviour rather than state.